rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -1049,17 +1052,23 @@ class Pickle private (settings: Settings, mtab: Mtab, sroot1: String, sroot2: St
     }
     def svalField(sgetterSym: String): s.SymbolInformation = {
       val noGetter = sgetterSym.isPrivateThis && !sgetterSym.isLazy
+      // In traits, a "getter" method is still emitted.
+      val method = sgetterSym.owner.isTrait && noGetter
```

Comment:
This is kind of confusing. Does `method` mean a getter method should be generated--or a getter method generated elsewhere should be used here? And yet `noGetter` is true at the same time? If so, please refactor the logic so that it doesn't seem contradictory.

Reply:
Great feedback -- allowed me to understand the principle of this more; the solution is much simpler.